website/common/script/content/hatching-potions.js

```diff
@@ -311,12 +319,14 @@ const premium = {
     value: 2,
     text: t('hatchingPotionAurora'),
     limited: true,
+    _addlNotes: t('eventAvailabilityReturning', {
+      availableDate: t('dateEndJanuary'),
+      previousDate: t('decemberYYYY', { year: 2019 }),
+    }),
+    event: EVENTS.winter2021,
     canBuy () {
-      return moment().isBetween('2019-12-19', '2020-02-02');
+      return moment().isBetween('2020-12-22T08:00-04:00', '2021-01-31T20:00-04:00');
```

Comment:
not now but maybe we could change these dates to use the event start and end dates directly? instead of having to declare them for each item

Reply:
In this specific case, the hatching potions come out on 12/22 which doesn't correspond to an event as such. But some of the other ones I may be able to fix, I'll take a look!